include/oneapi/internal/kernel_scheduler_sycl.h

```diff
@@ -141,10 +260,34 @@ class OpenClProgramRef : public OpenClResourceRef<cl_program, OpenClRetainProgra
         DAAL_CHECK_OPENCL(err, status)
     }
 
-    const char * getName() const { return _progamName.c_str(); }
+    void initModuleLevel0(ze_device_handle_t & zeDevice, services::Status * status = nullptr)
+    {
+        size_t binary_size = 0;
+        DAAL_CHECK_OPENCL(clGetProgramInfo(get(), CL_PROGRAM_BINARY_SIZES, sizeof(size_t), &binary_size, NULL), status);
+
+        auto binary = (unsigned char *)malloc(binary_size);
+        if (binary == nullptr) DAAL_CHECK_OPENCL(CL_OUT_OF_HOST_MEMORY, status);
```

Comment:
Use DAAL_CHECK_MALLOC

Reply:
used services::internal::tryAssignStatus(status, services::ErrorMemoryAllocationFailed);
due to DAAL_CHECK_MALLOC makes return